### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by traversaro**

### CODE REVIEW

**Comment by traversaro** [comment]
Given the absence of comments, I will merge as soon as the CIs are successful.

**Comment by jeljaik** [comment]
Looks good to me

File: README.md [hunk 1]
```diff
@@ -30,14 +42,14 @@ The `libraries` component contains librares developed by the CoDyCo consortium,
...
-* `yarp-wholebodyinterface`: Implementation of the wholeBodyInterface for YARP robots [Project Page](https://github.com/robotology-playground/yarp-wholebodyinterface)
+* [`codyco-commons`](https://github.com/robotology-playground/codyco-commons) : A collection of functions and utilities used in the other projects 
+* [`idyntree`](https://github.com/robotology/idyntree) : YARP-based Floating Base Robot Dynamics Library
+* [`paramHelp`](https://github.com/robotology-playground/paramHelp) : Library for simplifying the management of the parameters of YARP modules
+* [`wholebodyinterface`](https://github.com/robotology-playground/wholebodyinterface) : C++ Interfaces to sensor measurements, state estimations, kinematic/dynamic model and actuators for a floating base robot 
+* [`yarp-wholebodyinterface`](https://github.com/robotology-playground/yarp-wholebodyinterface) : Implementation of the wholeBodyInterface for YARP robots 
 * `orocosBFLBerdy`: Fork of the Bayesian Filtering Library [Orocos-BFL](http://www.orocos.org/bfl). *Early stage. Currently used and tested only for the quaternionEKF module and basic Kalman Filtering*
```
Comment (jeljaik) [inline README.md]: This needs to change into wholeBodyEstimator instead of quaterionEKF which I should have removed. If not, I will do it soon.

**Comment by traversaro** [comment]
Merging to fix https://github.com/robotology/WB-Toolbox/issues/10 .

**Pull request merged**